Sources/NimbleTests/Matchers/EqualTest.swift

```diff
@@ -32,7 +32,7 @@ class EqualTest: XCTestCase {
         expect(array1).to(equal([1, 2, 3]))
         expect(array1).toNot(equal([1, 2] as Array<Int>))
 
-        expect(NSArray(array: [1, 2, 3])).to(equal(NSArray(array: [1, 2, 3])))
```

Comment:
Perhaps it's better to just compiler-comment out this line? This is to array bridging from swift arrays <-> NSArrays